generated/Registrar/Registrar.ts

```diff
@@ -590,6 +548,48 @@ export class Registrar extends ethereum.SmartContract {
     return ethereum.CallResult.fromValue(value[0].toBoolean());
   }
 
+  isAvailable(id: BigInt): boolean {
+    let result = super.call("isAvailable", "isAvailable(uint256):(bool)", [
+      ethereum.Value.fromUnsignedBigInt(id)
+    ]);
+
+    return result[0].toBoolean();
+  }
+
+  try_isAvailable(id: BigInt): ethereum.CallResult<boolean> {
```

Comment:
naming of functions should be consistent,
instead of `try_isAvailable` this should be `tryIsAvailable`

Reply:
this was autogenerated by the graph coomand line. I didnt name it